utilities/src/main/java/io/syndesis/qe/bdd/validation/SftpValidationSteps.java

```diff
@@ -0,0 +1,34 @@
+package io.syndesis.qe.bdd.validation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.syndesis.qe.utils.SftpUtils;
+import io.syndesis.qe.utils.TestUtils;
+
+import java.util.concurrent.TimeUnit;
+
+import cucumber.api.java.en.Then;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class SftpValidationSteps {
```

Comment:
wouldn't it make more sense to have only one FtpValidationSteps class, since both ftpUtils class implement the same interface, so the methods should be the same

Reply:
Actually I had it this way originally. But later on I've separated it like this. My reason was: if you want to instantiate both FtpUtlils and SftpUtils at the beginning, you have to deploy both ftp/sftp servers. Why to do that, if only one is usually needed? 
There was also option to have general FileTransferUtils utils = null, and instantiate it either with FtpUtils or SftpUtils on demand (what takes some time) and switch between them, if you need them both. This option I found somehow not so elegant as separate steps. 
Buf If you still insist I can do that that way.

Reply:
ok, it can stay as it is